pytest_selenium/pytest_selenium.py

```diff
@@ -207,59 +200,32 @@ def pytest_addoption(parser):
                      metavar='str',
                      help='additional capability to set in format '
                           '"name:value".')
-    group._addoption('--chromepath',
+    group._addoption('--driver-path',
                      action='store',
-                     dest='chrome_path',
+                     dest='driver_path',
```

Comment:
We'll need to update the following to refer to the new option name:
https://github.com/justinpotts/pytest-selenium/blob/comline_options/pytest_selenium/driver.py#L69
https://github.com/justinpotts/pytest-selenium/blob/comline_options/pytest_selenium/driver.py#L79
https://github.com/justinpotts/pytest-selenium/blob/comline_options/pytest_selenium/driver.py#L97